package.json

```diff
@@ -51,6 +50,8 @@
     "typescript": "^4.1.4"
   },
   "dependencies": {
+    "@types/rimraf": "^3.0.0",
+    "@types/semver": "^7.3.4",
```

Comment:
Moving these shouldn't be required, the types of these packages are not re-exported as part of our defs?

Reply:
@MarshallOfSound I guess you right